components/list/list.html

```diff
@@ -0,0 +1,150 @@
+                    </li>
+                </ul>
+            </div>
+
+            <div>
+                <iframe id="frame"
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=61px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522list-container%2520ml3%2522%253E%250A%250A%2520%2520%2520%2520%2520%253Cul%2520class%253D%2522simple-list%2522%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%2520%253Cbutton%253EX%253C%252Fbutton%253Eyou%2520have%25202%2520messages%2520%253Cspan%2520class%253D%2522stack-list-time%2522%253E2hr%2520ago%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253E%253Cspan%2520class%253D%2522stack-list-sub%2522%253Etest%2520notification%253C%252Fspan%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%2520%253Cbutton%253EX%253C%252Fbutton%253Eyou%2520have%25202%2520messages%2520%253Cspan%2520class%253D%2522stack-list-time%2522%253E2hr%2520ago%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253E%253Cspan%2520class%253D%2522stack-list-sub%2522%253Etest%2520notification%253C%252Fspan%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%2520%253Cbutton%253EX%253C%252Fbutton%253Eyou%2520have%25204%2520messages%2520%253Cspan%2520class%253D%2522stack-list-time%2522%253E1hr%2520ago%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253E%253Cspan%2520class%253D%2522stack-list-sub%2522%253Etest%2520notification%253C%252Fspan%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%2520%253Cbutton%253EX%253C%252Fbutton%253Eyou%2520have%25201%2520message%2520%253Cspan%2520class%253D%2522stack-list-time%2522%253E30min%2520ago%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%253E%253Cspan%2520class%253D%2522stack-list-sub%2522%253Etest%2520notification%253C%252Fspan%253E%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Ful%253E%250A%2520%253C%252Fdiv%253E"
+                style=" height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
```

Comment:
it would be better if you can add a class for styling of Iframe, in that way the iframe style would be common throughout the code, and inline styling can be avoided. rest looks good.

Reply:
ok i will take care,thnks for the review